packages/@ama-sdk/client-fetch/schematics/ng-add/index.ts

```diff
@@ -0,0 +1,79 @@
+import { chain, noop, Rule } from '@angular-devkit/schematics';
+import type { NgAddSchematicsSchema } from './schema';
+import * as path from 'node:path';
+import { NodeDependencyType } from '@schematics/angular/utility/dependencies';
+
+const devDependenciesToInstall: string[] = [
+
+];
+
+
+const reportMissingSchematicsDep = (logger: { error: (message: string) => any }) => (reason: any) => {
+  logger.error(`[ERROR]: Adding @ama-sdk/client-fetch has failed.
```

Comment:
Is this package depends on `@o3r/core` ?

Reply:
No but the run of the schematics does.
This is the generic message we display when `@o3r/schematics` is missing